Sources/Swift/SentryExperimentalOptions.swift

```diff
@@ -1,7 +1,18 @@
 @objcMembers
 public class SentryExperimentalOptions: NSObject {
+    /**
+     * Disables swizzling of`NSData` to automatically track file operations.
+     *
+     * - Note: Swizzling is enabled by setting ``SentryOptions.enableSwizzling`` to `true`.
+     *         This option allows you to disable swizzling for `NSData` only, while keeping swizzling enabled for other classes.
+     *         This is useful if you want to use manual tracing for file operations.
+     */
+    public var disableDataSwizzling = false
```

Comment:
`h`: We don't have a single option starting with `disable`. Please change it to an enable flag and make the default true.
```suggestion
    public var enableDataSwizzling = true
```